ion-schema/src/system.rs

```diff
@@ -953,6 +989,13 @@ impl SchemaSystem {
             .load_schema(id, &mut TypeStore::default(), None)
     }
 
+    /// Requests each of the provided [`DocumentAuthority`]s, in order, to get ISL model for the
+    /// requested schema id until one successfully resolves it.
+    /// If an authority throws an exception, resolution silently proceeds to the next authority.
+    pub fn load_isl_schema<A: AsRef<str>>(&mut self, id: A) -> IonSchemaResult<IslSchema> {
+        self.resolver.load_isl_schema(id, None)
+    }
```

Comment:
Suppose I already have a `Rc<Schema>`, is there any way for me to get the `IslSchema` without having to reload it from the `Authority`s?

If I have an `IslSchema`, is there any way to go straight to `Rc<Schema>`?

Reply:
Currently there is no way to get `IslSchema` from `Rc<Schema>` or get Rc<Schema> from `IslSchema`. 
1. I will create another API that gets `Rc<Schema>` from an `IslSchema`
2. I will have to store `IslSchema` with the resolved `Rc<Schema>`  or create a store where the conversion is mapped. So that one can retrieve `IslSchema` from `Rc<Schema>`